Tests/LoadTests.swift

```diff
@@ -1,49 +1,48 @@
+import SlideController
 import UIKit
 import XCTest
-import SlideController
 
 class LoadTests: BaseTestCase {
-    
     func testLoadOnContentUnloadingEnabled() {
         slideController.isContentUnloadingEnabled = true
         let page1 = SlideLifeCycleObjectBuilder<TestableLifeCycleObject>(object: TestableLifeCycleObject())
         let page2 = SlideLifeCycleObjectBuilder<TestableLifeCycleObject>(object: TestableLifeCycleObject())
         let page3 = SlideLifeCycleObjectBuilder<TestableLifeCycleObject>(object: TestableLifeCycleObject())
         let givenContent = [page1, page2, page3]
         slideController.append(object: givenContent)
-        
+
         guard let object1 = page1.lifeCycleObject as? TestableLifeCycleObject,
             let object2 = page2.lifeCycleObject as? TestableLifeCycleObject,
             let object3 = page3.lifeCycleObject as? TestableLifeCycleObject else {
             XCTFail("page is not TestableLifeCycleObject")
             return
         }
-        
+
         XCTAssert(object1.viewDidLoadTriggered)
         XCTAssert(object2.viewDidLoadTriggered)
         XCTAssert(!object3.viewDidLoadTriggered)
     }
-    
+
     func testLoadOnContentUnloadingDisabled() {
         slideController.isContentUnloadingEnabled = false
         let page1 = SlideLifeCycleObjectBuilder<TestableLifeCycleObject>(object: TestableLifeCycleObject())
         let page2 = SlideLifeCycleObjectBuilder<TestableLifeCycleObject>(object: TestableLifeCycleObject())
         let page3 = SlideLifeCycleObjectBuilder<TestableLifeCycleObject>(object: TestableLifeCycleObject())
         let givenContent = [page1, page2, page3]
         slideController.append(object: givenContent)
-        
+
         guard let object1 = page1.lifeCycleObject as? TestableLifeCycleObject,
             let object2 = page2.lifeCycleObject as? TestableLifeCycleObject,
             let object3 = page3.lifeCycleObject as? TestableLifeCycleObject else {
-                XCTFail("page is not TestableLifeCycleObject")
-                return
+            XCTFail("page is not TestableLifeCycleObject")
+            return
```

Comment:
I'm not sure that guard with multiple conditions became more readable. Otherwise it makes harder to notice where `else` statements began

I would prefer to leave it as it was:
```swift
guard let object1 = page1.lifeCycleObject as? TestableLifeCycleObject,
    let object2 = page2.lifeCycleObject as? TestableLifeCycleObject,
    let object3 = page3.lifeCycleObject as? TestableLifeCycleObject else {
        XCTFail("page is not TestableLifeCycleObject")
        return
}
```
Or insert empty line between to clarify it
```swift
guard let object1 = page1.lifeCycleObject as? TestableLifeCycleObject,
    let object2 = page2.lifeCycleObject as? TestableLifeCycleObject,
    let object3 = page3.lifeCycleObject as? TestableLifeCycleObject else {

    XCTFail("page is not TestableLifeCycleObject")
    return
}
```